pymc/distributions/mixture.py

```diff
@@ -404,6 +403,38 @@ def marginal_mixture_logprob(op, values, rng, weights, *components, **kwargs):
     return mix_logp
 
 
+@_logcdf.register(MarginalMixtureRV)
+def marginal_mixture_logcdf(op, value, rng, weights, *components, **kwargs):
+
+    # single component
+    if len(components) == 1:
+        # Need to broadcast value across mixture axis
+        mix_axis = -components[0].owner.op.ndim_supp - 1
+        components_logcdf = logcdf(components[0], at.expand_dims(value, mix_axis))
+    else:
+        components_logcdf = at.stack(
+            [logcdf(component, value) for component in components],
+            axis=-1,
+        )
+
+    mix_logcdf = at.logsumexp(at.log(weights) + components_logcdf, axis=-1)
+
+    # Squeeze stack dimension
+    # There is a Aeasara bug in squeeze with negative axis
```

Comment:
```suggestion
    # There is a Aesara bug in squeeze with negative axis
```

Reply:
Maybe link to the issue so that we know when we can remove it.

Reply:
Done